fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+ font-family: 'pt_serif';
+ src: url('fonts/ptserif-bold-webfont.woff') format('woff');
+ font-weight: bold;
+ font-style: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6242821)